jwst/associations/tests/helpers.py

```diff
@@ -89,23 +88,6 @@ def test_run_generate(self):
                             )
 
 
-def make_megapool():
-    """
-    Combine the individual test pools into one.
-
-    Notes
-    -----
-    This is meant to be run in the source tree in
-    the folder the test pools reside in. The package
-    does need to have been installed.
-    `python -c 'import jwst.associations.tests.helpers as helpers; helpers.make_megapool()'`
-    """
-    pool_files = Path.glob("pool_*.csv")
-    pool_files.sort()
-    pool = combine_pools(pool_files)
-    pool.write("mega_pool.csv", format="ascii", delimiter="|", overwrite=True)
-
-
 # Basic utilities.
 def t_path(partial_path):
```

Comment:
Is this still needed, or can it be removed?

Reply:
It is still used in the standards regtests. I didn't touch those because the whole test file will be deleted in https://github.com/spacetelescope/jwst/pull/9520 anyway, so I can remove `t_path` in that PR when I get to it.